module/VuFind/src/VuFind/RecordTab/TOC.php

```diff
@@ -56,6 +56,6 @@ public function getDescription()
     public function isActive()
     {
         $toc = $this->getRecordDriver()->tryMethod('getTOC');
-        return !empty($toc);
+        return parent::isActive() || !empty($toc);
```

Comment:
I'd recommend reversing the order of these clauses -- parent::isActive could be an expensive operation, so if !empty($toc) evaluates to true, we want to be able to short-circuit out and avoid it.

Reply:
Sounds good!